Silicon/CommonSocPkg/Include/CpuInitDataHob.h

```diff
@@ -39,7 +42,8 @@ typedef struct {
   UINT8                  EnableItbm;                              /// Enable/Disable Intel Turbo Boost Max Technology 3.0.
   UINT8                  EnableItbmDriver;                        /// Enable/Disable Intel Turbo Boost Max Technology 3.0 Driver.
   EFI_PHYSICAL_ADDRESS   CpuConfigData;                           /// CPU RC Config for DXE consumption
-  EFI_PHYSICAL_ADDRESS   FvidTable;                               /// FVID Table
+  EFI_PHYSICAL_ADDRESS   CpuGnvsPointer;    ///< CPU_NVS_AREA Pointer.
+  EFI_PHYSICAL_ADDRESS   FvidTable;         ///< FVID Table.
 } CPU_INIT_DATA_HOB;
```

Comment:
CPU_INIT_DATA_HOB is produced by FSP, SBL just consume this HOB. It looks this structure is different for different platform, so SBL could not have a common structure.